tests/baselines/reference/tscWatch/projectsWithReferences/on-sample-project-with-nodenext.js

```diff
@@ -513,28 +513,10 @@ Resolving in CJS mode with conditions 'require', 'types', 'node'.
 Loading module as file / folder, candidate module location '/user/username/projects/sample1/core/anotherModule', target file types: TypeScript, JavaScript, Declaration.
 File '/user/username/projects/sample1/core/anotherModule.ts' exists - use it as a name resolution result.
 ======== Module name '../core/anotherModule' was successfully resolved to '/user/username/projects/sample1/core/anotherModule.ts'. ========
-File '/user/username/projects/sample1/core/package.json' does not exist.
-File '/user/username/projects/sample1/package.json' does not exist according to earlier cached lookups.
-File '/user/username/projects/package.json' does not exist according to earlier cached lookups.
-File '/user/username/package.json' does not exist according to earlier cached lookups.
-File '/user/package.json' does not exist according to earlier cached lookups.
-File '/package.json' does not exist according to earlier cached lookups.
-File '/user/username/projects/sample1/logic/package.json' does not exist.
-File '/user/username/projects/sample1/package.json' does not exist according to earlier cached lookups.
-File '/user/username/projects/package.json' does not exist according to earlier cached lookups.
-File '/user/username/package.json' does not exist according to earlier cached lookups.
-File '/user/package.json' does not exist according to earlier cached lookups.
-File '/package.json' does not exist according to earlier cached lookups.
```

Comment:
I haven’t been able to figure out why these lookups disappeared. It looks like all projects in the graph use `moduleResolution: nodenext`, so I'm not sure why any existing lookups would go away. Also debugged this test and never hit a breakpoint with `!shouldLookupFromPackageJson` in `getImpliedNodeFormatForFileWorker` 🤔

Reply:
Figured it out—the lookups didn’t go away; they were just silenced because the referenced project whose options we’re using to do the package.json lookups doesn’t have `traceResolution` set, while the downstream project does. I can fix that.